docs/UserGuide.md

```diff
@@ -19,26 +19,56 @@ By using our application, you can tag frequently used files/folders with a short
 with a single command. We hope by using our application, you can manage your files with ease. You can
 make your life easier one file at a time, and free up your precious time to spend on things you truly enjoy.
 
-## Table of Contents
+## 2. Table of Contents
+* [1. Introduction](#introduction)
+* [2. Table of Contents](#table-of-contents)
+* [3. Getting Started](#getting-started)
+    * [3.1 Installation](#installation)
+    * [3.2 Quick Start](#quick-start)
+    * [3.3 User Interface](#user-interface)
+    * [3.4 Basic Workflow](#basic-workflow)
+* [4. Features](#features)
+    * [4.1 Adding a tag with filepath: `tag`](#adding-a-tag-with-filepath--tag)
+    * [4.2 Displaying information of a tagged file: `show`](#displaying-information-of-a-tagged-file--show)
+    * [4.3 Accessing a tagged file : `open`](#accessing-a-tagged-file--open)
+    * [4.4 Removing a tag: `untag`](#removing-a-tag--untag)
+    * [4.5 Renaming a tag: `retag`](#renaming-a-tag--retag)
+    * [4.6 Adding a label to a tag : `label`](#adding-a-label-to-a-tag--label)
+    * [4.7 Deleting multiple labels from a tag : `unlabel`](#deleting-multiple-labels-from-a-tag--unlabel)
+    * [4.8 Finding a tag: `tag`](#finding-a-tag--find)
+    * [4.9 Changing current directory: `cd`](#changing-current-directory--cd)
+    * [4.10 Listing all tags: `list`](#listing-all-tags--ls)
+    * [4.11 Undoing command: `undo`](#undoing-command--undo)
+    * [4.12 Redoing command: `redo`](#redoing-command--redo)
+    * [4.13 Clearing all tags: `clear`](#clearing-all-tags--clear)
+    * [4.14 Exiting the application: `exit`](#exiting-the-application--exit)
+    * [4.15 Viewing the help: `help`](#viewing-help--help)
+* [5. Customisation](#customisation)
+    * [5.1 Changing themes](#changing-themes)
+    * [5.2 Changing view size](#changing-view-sizes)
+    * [5.3 Persistent customisation](#persistent-customisation)
+* [6. FAQ](#faq)
+* [7. Glossary](#glossary)
+* [8. Command Summary](#command-summary)
 * Table of Contents
 {:toc}
```

Comment:
You can remove the above 2 lines since we have manually added the table of contents.